File: copy_layout.py
import re

with open('src/components/LabelDashboardViewV2.tsx', 'r') as f:
    lines = f.readlines()

end_idx = 0
for i, line in enumerate(lines):
    if "Row 5: ROSTER GRID" in line or "Quick Action Buttons (Desktop)" in line or "Quick Action Buttons" in line:
        end_idx = i
        break

print(f"End index: {end_idx}")

File: copy_layout.js
const fs = require('fs');

const lines = fs.readFileSync('src/components/LabelDashboardViewV2.tsx', 'utf-8').split('\n');

let endIdx = 0;
for (let i = 0; i < lines.length; i++) {
    if (lines[i].includes("Quick Action Buttons (Desktop)")) {
        endIdx = i;
        break;
    }
}
console.log("End index: " + endIdx);
